.github/workflows/test.yml

```diff
@@ -52,12 +50,18 @@ jobs:
         poetry env use "${{ matrix.python-version }}"
         poetry install
 
-    - name: Run pytest
+    - name: Run pytest linux
+      if: (matrix.os != 'windows-latest')
       run: |
         poetry run coverage run --parallel -m pytest -m "$PYTEST_MARKERS"
+    
+    - name: Run pytest windows
+      if: (matrix.os == 'windows-latest')
+      run: |
+        poetry run coverage run --parallel -m pytest -m $Env:PYTEST_MARKERS
 
     - name: Upload coverage data
-      if: always() && (matrix.python-version == '3.9')
+      if: always() && (matrix.python-version == '3.9') && (matrix.os == 'ubuntu-latest')
```

Comment:
@aaronsteers any reason we don't want to combine coverage from multiple Python versions? If not, I guess there's also no reason not to combine coverage from multiple OSs. Wdyt?

> Often test suites are run under different conditions, for example, with different versions of Python, or dependencies, or on different operating systems. In these cases, you can collect coverage data for each test run, and then combine all the separate data files into one combined file for reporting.

https://coverage.readthedocs.io/en/latest/cmd.html#cmd-combine

Reply:
@edgarrmondragon and @visch - No reasons I'm aware to collect and combine coverage. This snippet was written before we had a combine step (or before I understood it. 😄 )

We can open this as a separate issue though rather than delay to PR.